src/Admin/SystemStatusService.php

```diff
@@ -0,0 +1,156 @@
+<?php
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\GoogleListingsAndAds\Admin;
+
+use Automattic\WooCommerce\GoogleListingsAndAds\API\WP\NotificationsService;
+use Automattic\WooCommerce\GoogleListingsAndAds\Infrastructure\Registerable;
+use Automattic\WooCommerce\GoogleListingsAndAds\Infrastructure\Service;
+use Exception;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class SystemStatusService
+ * This class adds Google for WooCommerce information to the WooCommerce System Status Report
+ *
+ * @since 3.2.0
```

Comment:
Using a version placeholder would be a bit nicer.

```suggestion
 * @since x.x.x
```